src/auth/authStore.js

```diff
@@ -57,6 +57,16 @@ export default class AuthStore {
     }
   }
 
+  removeUserFromOrg = async ({ organizationId, userId, claims, updateClaims = claims => claims }) => {
```

Comment:
Don't allow removeUserFromOrg to modify other claims or you've introduced an undocumented functionality to it.

Reply:
In fact, it doesn't look like this code even removes a user from an organization...